.github/workflows/test-mobile-e2e-reusable.yml

```diff
@@ -252,6 +252,35 @@ jobs:
           role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID_PROD }}:role/${{ secrets.AWS_CACHE_ROLE_NAME }}
           aws-region: ${{ secrets.AWS_CACHE_REGION }}
 
+      - name: Setup iOS timing cache
+        timeout-minutes: 5
+        uses: tespkg/actions-cache/restore@v1
+        with:
+          path: e2e/mobile/artifacts/e2e-test-results-ios.json
+          key: "${{ needs.determine-builds.outputs.ios_timing_cache_key }}"
```

Comment:
I see that timing files cache key file depends on the specs folder. So you confim that we don't get any split based on timing each time we add a new test ?